contributing/ORIGIN_TRIALS.md

```diff
@@ -0,0 +1,22 @@
+# Origin trials
```

Comment:
Nit: Let's consistently capitalize as "Origin Trials" in section headers and "origin trials" in content text.